aten/src/ATen/Context.h

```diff
@@ -72,6 +73,8 @@ class TORCH_API Context {
       return at::detail::getHIPHooks();
     } else if (opt_device_type == at::kHPU) {
       return at::detail::getHPUHooks();
+    } else if (opt_device_type == at::kXLA) {
+      return at::detail::getXLAHooks();
```

Comment:
I think you miss XLA in https://github.com/pytorch/pytorch/blob/21697feff257ad04dd916ef63b8b841c38f7e9ee/aten/src/ATen/DeviceAccelerator.cpp#L43-L47 if you would like to add XLA to accelerator.

Reply:
Otherwise, you never correctly got XLA from `getAcceleratorHooksInterface`. I don't know whether XLA should be treated as an accelerator yet.